### ISSUE TITLE
Benchmarking of FASTA file

### ISSUE THREAD

**Issue by CamiloPosso** [microbiomedata/kaiko_metaproteome#5]
Use X-Tandem quick search to test how well raw spectra protein search performs with a particular taxa.

**Comment by CamiloPosso** [comment]
Still working on this one, should be done this week

**Comment by ssarrafan** [comment]
@CamiloPosso is this still in progress?  I'm checking squad boards and seeing this so will move back to the current sprint. Let me know if you're not working on it.

**Comment by ssarrafan** [comment]
@CamiloPosso @pdpiehowski what's the status of this? Can it be closed yet?

**Comment by CamiloPosso** [comment]
I'm moving this to backlog, since we are prioritizing the deployment of Kaiko 1.0. Thanks

**Comment by ssarrafan** [comment]
@CamiloPosso @pdpiehowski is this issue still on your radar?  Is this still in the plan?

**Comment by CamiloPosso** [comment]
Yes we will be working on this. I added an issue with more updated information and will close this one as a duplicate. Thanks @ssarrafan
